### DISCUSSION TITLE
Remove redundant code

### DISCUSSION THREAD

**Discussion by n-at-han-k** [glaucocustodio/tanakai#11] [Ideas]
I was looking through the code and noticed that there’s a private method that is never called within the class.
As far as I can tell anyway.
Should this be removed to clean up the code?

[Tanakai::Base#create_brower](https://github.com/glaucocustodio/tanakai/blob/0df7cf580dc5ed36e0a8e4e7f031b6c72780d932/lib/tanakai/base.rb#L253C1-L255C8)

**Comment by glaucocustodio** [comment]
sure, I would accept a PR removing that :), thank you.
